multiuser/keycloak/che-multiuser-keycloak-user-remover/src/main/java/org/eclipse/che/multiuser/keycloak/server/KeycloakTokenRequester.java

```diff
@@ -0,0 +1,89 @@
+/*
+ * Copyright (c) 2012-2021 Red Hat, Inc.
+ * This program and the accompanying materials are made
+ * available under the terms of the Eclipse Public License 2.0
+ * which is available at https://www.eclipse.org/legal/epl-2.0/
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ *
+ * Contributors:
+ *   Red Hat, Inc. - initial API and implementation
+ */
+package org.eclipse.che.multiuser.keycloak.server;
+
+import static java.nio.charset.StandardCharsets.UTF_8;
+import static javax.ws.rs.HttpMethod.POST;
+
+import com.google.gson.JsonElement;
+import com.google.gson.JsonObject;
+import com.google.gson.JsonParser;
+import com.google.gson.JsonSyntaxException;
+import java.io.BufferedReader;
+import java.io.IOException;
+import java.io.InputStreamReader;
+import java.io.OutputStream;
+import java.net.HttpURLConnection;
+import java.net.URL;
+import javax.ws.rs.core.HttpHeaders;
+import javax.ws.rs.core.MediaType;
+import org.eclipse.che.api.core.ServerException;
+import org.eclipse.che.commons.lang.IoUtil;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+public class KeycloakTokenRequester {
```

Comment:
should be `@Singleton` + could you please add a relevant javadoc for the class
also do not hesitate to add yourself as @author ;-)